packages/browser-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bucketco/browser-sdk",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
are thy ready for prime time? Should we prefix them all with `alpha.x` instead?

Reply:
they are ready. Also, it's too late, already shipped 1.0 ;)